posthog-android/src/main/java/com/posthog/android/replay/PostHogReplayIntegration.kt

```diff
@@ -729,10 +729,11 @@ public class PostHogReplayIntegration(
             // we have to copy without the childWireframes, otherwise they all would be different
             // if one of the child is different, but we only wanna compare the parent
```

Comment:
I guess the comment is out of date now... otherwise this saves me so much work 🙈

Reply:
The comment is for the comparison, so it's still up to date.
Instead of comparing property per property, I compare the hashCode so I know "something" has changed.